sdk/python/feast/job_service.py

```diff
@@ -145,6 +167,54 @@ def GetJob(self, request, context):
         return GetJobResponse(job=self._job_to_proto(job))
 
 
+def run_control_loop():
+    """Runs control loop that continuously ensures that correct jobs are being run."""
+    logging.info(
+        "Feast Job Service is starting a control loop in a background thread..."
+    )
+    client = feast.Client()
+    while True:
+        # Map of job hash -> (project, table_name)
+        table_metadata: Dict[str, Tuple[str, str]] = dict()
```

Comment:
We try and avoid using the word `metadata` since its typically used when people cant think of a good name. Can we be more descriptive with this variable? `job_hash_to_table_references`?

Reply:
How about job_hash_to_table_refs to make it little shorter?